app/controllers/CheckYourAnswersController.scala

```diff
@@ -41,12 +42,111 @@ class CheckYourAnswersController @Inject()(
     implicit request =>
 
       val helper = new CheckYourAnswersHelper(request.userAnswers)
+      val isOrganisation: Boolean = isOrganisationJourney(request.userAnswers)
+      val businessDetails: Seq[SummaryList.Row] = buildDetails(helper, isOrganisation)
+      val contactDetails: Seq[SummaryList.Row] = buildContactDetails(helper)
 
-      val answers: Seq[SummaryList.Row] = Seq()
+      val header: String = if (isOrganisation) {
+        "checkYourAnswers.businessDetails.h2"
+      } else {
+        "checkYourAnswers.individual.h2"
+      }
 
       renderer.render(
         "check-your-answers.njk",
-        Json.obj("list" -> answers)
+        Json.obj(
+          "header" -> header,
+          "businessDetailsList" -> businessDetails,
+          "contactDetailsList" -> contactDetails,
+        )
       ).map(Ok(_))
   }
+
+  private def buildDetails(helper: CheckYourAnswersHelper, isOrganisation: Boolean): Seq[SummaryList.Row] = {
+
+    val pagesToCheck = Tuple4(
+      helper.businessType,
+      helper.nino,
+      helper.businessWithoutIDName,
+      helper.nonUkName
+    )
+
+    pagesToCheck match {
+      case (Some(_), None, None, None) if isOrganisation =>
+        //Business with ID
+        Seq(
+          helper.confirmBusiness
+        ).flatten
+
+        //TODO Fix once Sole trader journey is working
+//      case (Some(_), None, None, None) =>
+//        //Sole trader
+//        Seq(
+//          ???
+//        ).flatten
+
+      case (None, Some(_), None, None) =>
+        //Individual with ID
+        Seq(
+          helper.doYouHaveUTR,
+          helper.doYouHaveANationalInsuranceNumber,
+          helper.nino,
+          helper.namePage,
+          helper.dateOfBirth
+        ).flatten
+      case (None, None, Some(_), None) =>
+        //Business without ID
+        Seq(
+          helper.doYouHaveUTR,
+          helper.registrationType,
+          helper.businessWithoutIDName,
+          helper.businessAddress
+        ).flatten
+      case (None, None, None, Some(_)) =>
+        //Individual without ID
+        Seq(
+          helper.doYouHaveUTR,
+          helper.doYouHaveANationalInsuranceNumber,
+          helper.nonUkName,
+          helper.dateOfBirth,
+          helper.doYouLiveInTheUK,
+          helper.whatIsYourAddress,
+          helper.whatIsYourAddressUk
+        ).flatten
+      case _ =>
+        //All pages
+        Seq(
+          helper.businessType,
+          helper.doYouHaveUTR,
+          helper.confirmBusiness,
+          helper.nino,
```

Comment:
Does business enter a nino?

Reply:
They don't. This case is for all possible pages that a user can get to just in case they jump straight to this page.

Reply:
Ok